beacon-chain/execution/check_transition_config.go

```diff
@@ -143,7 +143,8 @@ func (s *Service) logTtdStatus(ctx context.Context, ttd *uint256.Int) (bool, err
 	log.WithFields(logrus.Fields{
 		"latestDifficulty":   latestTtd.String(),
 		"terminalDifficulty": ttd.ToBig().String(),
-	}).Info("terminal difficulty has not been reached yet")
+		"network":            params.BeaconConfig().ConfigName,
+	}).Info("Ready for merge")
```

Comment:
```suggestion
	}).Info("Ready for The Merge. Details: ")
```

Feel free to ignore this suggestion if it doesn't seem like an improvement :)